src/core/workspace/folder-to-project.ts

```diff
@@ -96,6 +103,19 @@ function _processSourceFile(sourcePath: string, destFolder: string) {
     Package.extractResources<RawResource>(buffer, {
       loadRaw: true,
       decompressBuffers: true,
+      resourceFilter(type, group, inst) {
+        return type in TuningResourceType;
+      }
+    }).forEach(entry => {
+      _processResource(entry.key, entry.value.buffer, packageDest);
+    });
+
+    Package.extractResources<RawResource>(buffer, {
+      loadRaw: true,
+      decompressBuffers: true,
+      resourceFilter(type, group, inst) {
+        return !(type in TuningResourceType);
+      }
     }).forEach(entry => {
```

Comment:
I see why this is needed, because tuning needs to have its names mapped first, maybe just leave a quick comment though cause I wasn't sure about the purpose of this at first, and can see myself or someone else getting confused and reducing this down to one loop in the future.